jwst/lib/utc_to_tdb.py

```diff
@@ -121,15 +189,12 @@ def find_hdu(fd, extname):
     for i in range(1, nhdu):
         hdr = fd[i].header
         if "EXTNAME" in hdr and hdr["EXTNAME"] == extname_uc:
-            if hdunum is not None:
-                fd.close()
-                raise RuntimeError("There are at least two HDUs with "
-                                   "EXTNAME = {}; there should only be one."
-                                   .format(extname_uc))
-            hdunum = i 
+            hdunum = i
+            break
```

Comment:
With this `break` added here it looks like this function returns the index of he first extension with EXTNAME found in the file and it doesn't raise an error if there are more than one extensions with this EXTNAME. If so I wonder if
it's not faster to use `fd.index_of(EXTNAME)` instead of this function.

Reply:
Yes, I need to take another look at this function.  Originally it was only for finding the INT_TIMES extension, and there shouldn't be more than one.  Now it's also used for finding a SCI extension, and it should not be any problem if there's more than one, though I would be a little surprising.